internal/httputil/certlog.go

```diff
@@ -0,0 +1,157 @@
+package httputil
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/go-logr/logr"
+)
+
+const (
+	defaultLogLevel = 4
+)
+
+// Log the certificates that would be used for docker pull operations
+// Assumes a /etc/docker/certs.d like path, where the directory contains
+// <hostname>:<port> directories in which a CA certificate (generally
+// named "ca.crt") is located.
+func LogDockerCertificates(path string, log logr.Logger) {
+	var paths []string
+	if path == "" {
+		paths = []string{"/etc/docker/certs.d", "/etc/containers/certs.d"}
```

Comment:
`/etc/docker/certs.d` is used by docker for VA certs and `/etc/containers/certs.d` is used by Podman. Can we add a comment mentioning this as reason we look in to these two paths?

Reply:
Sure.